spark/src/main/java/org/apache/iceberg/spark/SparkFixupTimestampType.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.spark;
+
+import org.apache.iceberg.Schema;
+import org.apache.iceberg.types.FixupTypes;
+import org.apache.iceberg.types.Type;
+import org.apache.iceberg.types.TypeUtil;
+import org.apache.iceberg.types.Types;
+
+/**
+ * By default Spark TimestampType should be converted to Types.TimestampType.withZone() iceberg type.
+ * But we also can convert TimestampType to Types.TimestampType.withoutZone() iceberg type
+ * by setting SparkUtil.STORE_TIMESTAMP_WITHOUT_TIMEZONE_SESSION_PROPERTY to 'true'
```

Comment:
Using a {@link class#parameter} thing here would keep this in sync with the variable name 
Same note for referring to for time stamp types

Reply:
good point, fixed